third_party/skymarshal/cmake/skymarshal.cmake

```diff
@@ -54,7 +54,7 @@ function(add_cpp_bindings
     list(APPEND generated_files ${bindings_dir}/cpp/lcmtypes/${package}/${type}.hpp)
   endforeach()
 
-  add_library(${target_name}_cpp INTERFACE ${generated_files})
```

Comment:
Ah ok, is this why that target was missing when `SYMFORCE_USE_EXTERNAL_LCM=OFF`?

Reply:
That's my guess yeah (although I haven't actually reproduced and checked)